src/Frontend/Core.hs

```diff
@@ -454,6 +458,51 @@ ideasFilterQuery = \case
     (Just cat) -> filter ((== Just cat) . view ideaCategory)
     Nothing    -> id
 
+type IdeasSortApi = QueryParam "sortby" SortIdeasBy
+type IdeasSortQuery = Maybe SortIdeasBy
+
+data SortIdeasBy = SortIdeasByAge | SortIdeasBySupport
+  deriving (Eq, Ord, Show, Read, Enum, Bounded)
+
+instance FromHttpApiData SortIdeasBy where
+    parseUrlPiece = \case
+        "age" -> Right SortIdeasByAge
+        "sup" -> Right SortIdeasBySupport
+        _     -> Left "no parse"
+
+instance ToHttpApiData SortIdeasBy where
+    toUrlPiece = \case
+        SortIdeasByAge     -> "age"
+        SortIdeasBySupport -> "sup"
+
+ideasSortQuery :: IdeasSortQuery -> [Idea] -> [Idea]
+ideasSortQuery = f . fromMaybe minBound
+  where
+    f SortIdeasByAge     = age
```

Comment:
> Why not `age . sup` then?

i was thinking that, too, but i don't think there will ever be two ideas created in the same milisecond.  and if they are, i can live with the fact that their order is not specified.